content/webapp/pages/homepage.js

```diff
@@ -24,10 +51,14 @@ const pageImage =
   'https://images.prismic.io/wellcomecollection/fc1e68b0528abbab8429d95afb5cfa4c74d40d52_tf_180516_2060224.jpg?auto=compress,format&w=800';
 export class HomePage extends Component<Props> {
   static getInitialProps = async (ctx: Context) => {
-    const articles = await getArticles(ctx.req, { pageSize: 4 });
-    if (articles) {
+    const { id } = ctx.query;
+    const articlesPromise = getArticles(ctx.req, { pageSize: 4 }); // TODO make client side?
+    const pagePromise = await getPage(ctx.req, id);
+    const [articles, page] = await Promise.all([articlesPromise, pagePromise]);
```

Comment:
This should be a temporary solution. If we would like to start making the homepage have loads of calls, we should architect it to be able to do it in one call.

I know cache is there, but that should not be a reason to be building against common good accessibility guidelines.